platform/debian/systemd/kanidmd.service

```diff
@@ -7,14 +7,27 @@ After=network-online.target
 Wants=network-online.target
 
 [Service]
-Type=simple
+Type=notify
 
```

Comment:
Unlike in the Suse version, this would run as root, which is probably not ideal. I'd add a `User=kanidm` here and have the packagers deal with that. You could also add that to the Suse version, since even with `DynamicUser=yes` a static user of a configured name will be used if it exists, which is why it should be fine to just add `DynamicUser=yes` here as well.

Reply:
Yep, I'll copy the suse content here.